### ISSUE TITLE
⌨ Insufficient contrast for a few things

### ISSUE THREAD

**Issue by vd1992** [GCTC-NTGC/gc-digital-talent#10918]
## ⌨ Accessibility Issue

Contrast ratio a little low for a few things

### WCAG 2.1 Criteria

Meet the contrast threshold

## 📋 Steps to Reproduce

1. Observe the places below

## 📸 Screenshot

![346677718-4336a33a-efde-4c8d-9f22-0d05d93bf27c](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/bff4c562-44f3-4f42-83cc-e6f8cc4c8ce2)

![346685695-a0fea3c4-1233-4d5b-b091-1bb66cee5c30](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/b99334c2-7e9d-4ccb-9794-77122dba6999)

## 🙋‍♀️ Proposed Solution

Boost the contrast

## ✅ Acceptance Criteria

- [ ] use `error.dark` and `warning.darker`
- [ ] both light mode and dark mode should work nicely on the contrast level 
- [ ] Issue fixed in light mode and dark mode already working but shouldn't be impacted by this change.
